integration-cli/docker_cli_build_test.go

```diff
@@ -7292,3 +7297,25 @@ func (s *DockerSuite) TestBuildWindowsEnvCaseInsensitive(c *check.C) {
 		c.Fatalf("Case insensitive environment variables on Windows failed. Got %s", res)
 	}
 }
+
+// GETENV on Windows #29048
+func (s *DockerSuite) TestBuildWindowsGetenv(c *check.C) {
+	testRequires(c, DaemonIsWindows)
+	name := "testbuildwindowsgetenv"
+	_, out, err := buildImageWithOut(name, `
+		FROM `+WindowsBaseImage+`
+		GETENV PATH
+		ENV PATH c:\\something;$PATH
+		RUN echo Result:%PATH%
+  `, true)
+	if err != nil {
+		c.Fatal(err)
+	}
+	c.Assert(strings.ToLower(out), checker.Contains, `result:c:\something;`)
+	c.Assert(strings.ToLower(out), checker.Contains, `c:\windows\system32`)
```

Comment:
rather than having to split this check across two line, can you do this test in the Dockerfile, something like?
```
RUN [ "$PATH" = 'c:\\something;c:\\windows\\system32' ]
```